frontend/src/container/QueryBuilder/filters/QueryBuilderSearch/utils.ts

```diff
@@ -102,3 +102,7 @@ export function replaceStringWithMaxLength(
 	if (lastSearchValue === '') return `${mainString}${replacementString},`; // if user select direclty from options
 	return mainString.replace(lastSearchValue, `${replacementString},`);
 }
+
+export function checkCommaInValue(str: string): string {
+	return str.includes(',') ? `"${str}"` : str;
+}
```

Comment:
Please do not use the basic string functions. Please avoid doing that for non-trivial tasks. Why can't you use unparse here?

Reply:
This is just more of a utility function, it's just checking whether the string(Value) has a comma or not. @srikanthccv 
CC: @techchintan

Reply:
@srikanthccv @palashgdev 
input: ['**chintan**','**HTTP GET,GET,PUT**','**Sudani,POST**','**Super,"luxurious",truck**']
output: 
![image](https://github.com/SigNoz/signoz/assets/46838508/5b82279d-4f7b-4bf5-89e9-fddd787bda15)

Is this scenario right?